scripts/skills/perks/perk_rf_fresh_and_furious.nut

```diff
@@ -55,12 +55,25 @@ this.perk_rf_fresh_and_furious <- ::inherit("scripts/skills/skill", {
 			});
 		}
 
-		tooltip.push({
-			id = 12,
-			type = "text",
-			icon = "ui/icons/warning.png",
-			text = ::Reforged.Mod.Tooltips.parseString("Becomes disabled when starting a turn with " + ::MSU.Text.colorizeFraction(this.m.FatigueThreshold) + " or more [Fatigue|Concept.Fatigue] built")
-		});
+		// For non-dummy actor we also add the actual fatigue value calculated from the threshold
+		if (this.getContainer().getActor().getID() == ::MSU.getDummyPlayer().getID())
+		{
+			tooltip.push({
+				id = 12,
+				type = "text",
+				icon = "ui/icons/warning.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Becomes disabled when starting a turn with " + ::MSU.Text.colorizeFraction(this.m.FatigueThreshold) + " or more [Fatigue|Concept.Fatigue] built")
+			});
+		}
+		else
+		{
+			tooltip.push({
+				id = 12,
+				type = "text",
+				icon = "ui/icons/warning.png",
+				text = ::Reforged.Mod.Tooltips.parseString(format("Becomes disabled when starting a turn with %s (%s) or more [Fatigue|Concept.Fatigue] built", ::MSU.Text.colorizeFraction(this.m.FatigueThreshold), ::MSU.Text.colorRed(::Math.round(this.m.FatigueThreshold * this.getContainer().getActor().getFatigue()))))
```

Comment:
The first value is colored green while value in brackets is colored red. You should use the same color on both.

![image](https://github.com/Battle-Modders/mod-reforged/assets/2252464/5e560f8d-b6d5-4181-a401-22a65ade79d3)

In the perk tooltip we use red color for the threshold. Here we used green color previously.
I'd suggest `colorNeutral` if it was an option. Otherwise just choose one of the two colors for all

Reply:
It's supposed to be red. If it was being shown green previously that's unintended. Will fix.